src/harness/unittests/jsDocParsing.ts

```diff
@@ -305,6 +305,12 @@ namespace ts {
  * @param x hi
 < > still part of the previous comment
  */`);
+
+                parsesCorrectly("Nested @param tags",
+`/**
+* @param {object} o Doc doc
+* @param {string} o.f

```

Comment:
I thought it was the comments on nested `@param` that were incorrectly skipped. Shouldn’t this test include some comments for this line too?

Reply:
`parseChildParameterOrPropertyTag` *begins* by skipping a line. So it's the comment that comes *before* the child that was skipped. But I'll add some comments for that line too.